react/IOMessageWithMarkers.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/display-name */
+import React, { useMemo } from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+
+import IOMessage from './IOMessage'
+import { IOMessageWithMarkers as IOMessageWithMarkersType } from './typings/IOMessageWithMarkers'
```

Comment:
`.d.ts` files should not be directly imported. The project's `tsconfig.json` already includes all `.d.ts` inside the `typings` directory, you just need to use the type as if it's a global type.